src/odemis/driver/test/technolution_test.py

```diff
@@ -51,7 +51,7 @@
 logging.getLogger().setLevel(logging.DEBUG)
 
 # Export TEST_NOHW = 1 to prevent using the real hardware
-TEST_NOHW = (os.environ.get("TEST_NOHW", "0") != "0")  # Default to Hw testing
+TEST_NOHW = (os.environ.get("TEST_NOHW", "0") != "1")  # Default to Hw testing
```

Comment:
Now you changed it to no hardware testing by default: if TEST_NOHW does not exist `os.environ.get("TEST_NOHW", "0") = "0"` and `("0" != "1") = True`

Reply:
Indeed that should not be like this. I forgot to change it back.